site/source/tutorials/005_form_widgets/index.md

```diff
@@ -103,7 +103,7 @@ We are importing the `Button` class that will be used to provide the form's subm
 				type: 'email',
 				required: true
 			}),
-			w(Button, { content: 'Save' })
+			w(Button, {}, ['Save!'])
```

Comment:
Array literals should have spaces between the brackets and contents per the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md#braces-brackets-and-parentheses)

Reply:
Fixed!